package.json

```diff
@@ -20,7 +21,13 @@
     "eslint": "^8.45.0",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.3",
+    "husky": "^8.0.3",
     "typescript": "^5.0.2",
     "vite": "^4.4.5"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "chmod +x ./husky/commit-msg && bash ./husky/commit-msg"
```

Comment:
Setting permissions is a one time command. And file permissions are also saved in git.

Remove chmod from here, and run it on the file and make a commit

Reply:
ok